security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -209,40 +233,27 @@ func GetSecretName(saName string) string {
 
 // Determine if the object is "enabled" for Istio.
 // Currently this looks at the list of watched namespaces and the object's namespace annotation
-func (sc *SecretController) istioEnabledObject(obj metav1.Object) bool {
-	if _, watched := sc.namespaces[obj.GetNamespace()]; watched || !sc.explicitOptIn {
+func (sc *SecretController) istioManagedObject(obj metav1.Object) bool {
```

Comment:
Doesn't make sense - an object can be managed by Istio, just not by this particular instance.

Reply:
Agreed, renamed to `citadelManagedObject`.